### PR TITLE
PWA: Actually use new amp-shadow.js runtime

### PR DESCRIPTION
**Pull request by dvoytenko**
This for now assumes local distribution only. We will support compiled/cdn later.

### CODE REVIEW

File: examples/pwa/pwa.js [hunk 1]
```diff
@@ -315,23 +318,36 @@ class AmpViewer {
...
 
   /**
+   * @param {string} src
+   * @param {string=} customElement
+   * @param {string=} customTemplate
+   */
+  installScript_(src, customElement, customTemplate) {
+    const doc = this.win.document;
+    const el = doc.createElement('script');
+    el.setAttribute('async', '');
```
Comment (cramforce) [inline examples/pwa/pwa.js]: Not needed.

Comment (dvoytenko) [inline examples/pwa/pwa.js]: Removed

**Comment by dvoytenko** [comment]
@cramforce PTAL

**Comment by cramforce** [comment]
LGTM

**Pull request merged**